sysquant/estimators/stdev_estimator.py

```diff
@@ -159,4 +156,4 @@ def annualise_stdev_estimate(stdev: stdevEstimates, frequency: str) -> stdevEsti
 def annualised_stdev(stdev_value: float, frequency: str):
     how_many_times_a_year = how_many_times_a_year_is_pd_frequency(frequency)
 
-    return stdev_value * (how_many_times_a_year**.5)
\ No newline at end of file
+    return stdev_value * (how_many_times_a_year**.5)
```

Comment:
formatting

Reply:
Do you mean the newline at the end of the file? I wouldn't want to start a "holy war", but I believe that it is quite standard to terminate files with a newline (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
No, I'm not bothered about formatting, it's just a note to myself that the edit is only formatting and doesn't affect functionality. Sorry fot the confusion.